src/ctim/schemas/common.clj

```diff
@@ -305,22 +348,22 @@
    "Wrote_To" "Specifies that this object wrote to the related object."
    "Created" "Specifies that this object created the related object."})
 
-(s/defschema RelationType
-  (apply s/enum (keys relations-map)))
+(s/defschema ObservableRelationType
+  (apply s/enum (keys observable-relations-map)))
 
-(s/defschema Relation
+(s/defschema ObservableRelation
   {:id s/Num
    :timestamp Time
    :origin s/Str
    (s/optional-key :origin_uri) URI
-   :relation RelationType
+   :relation ObservableRelationType
    (s/optional-key :relation_info) {s/Keyword s/Any}
```

Comment:
the `s/Any` here pose problems at schema and code generation. For example, the haskell client can't be generated correctly because of it and also, the testing generators can't use the :relation key because they cause coercion errors.

Could we use something like `s/Str` or `{s/Keyword s/Str}`